DOC-2691-incremental backup [4.2] (#804)
tigergraph/server-docs: modules/backup-and-restore/pages/online-backup.adoc

== Limitations and Considerations

Currently, the Online Backup feature has one limitation:

> * If users start running a query, the full backup cannot succeed until that query finishes.

Reviewer: ```suggestion
* a query starts during a *full* backup, the backup will not finish until that query finishes.
```

> However, the *incremental backup* does not have this limitation. Users can perform incremental backups even if a query is running.

Reviewer: ```suggestion
However, *incremental backup* does not have this limitation. Users can perform incremental backups even if a query is running.
```
